<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="bg">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 85, 127);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="labelData">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 15px;
font: 11pt &quot;Myanmar Text&quot;;
color: rgb(255, 255, 255);
font-weight: bold;
</string>
      </property>
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="caixaData">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="caixaNome">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNome">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 15px;
font: 11pt &quot;Myanmar Text&quot;;
color: rgb(255, 255, 255);
font-weight: bold;
</string>
      </property>
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelOri">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 15px;
font: 11pt &quot;Myanmar Text&quot;;
color: rgb(255, 255, 255);
font-weight: bold;
</string>
      </property>
      <property name="text">
       <string>Origem</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="caixaOri">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelDest">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>180</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 15px;
font: 11pt &quot;Myanmar Text&quot;;
color: rgb(255, 255, 255);
font-weight: bold;
</string>
      </property>
      <property name="text">
       <string>Destino</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="caixaDest">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPagamento">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 15px;
font: 11pt &quot;Myanmar Text&quot;;
color: rgb(255, 255, 255);
font-weight: bold;
</string>
      </property>
      <property name="text">
       <string>Tipo de pagamento</string>
      </property>
     </widget>
     <widget class="QComboBox" name="caixaPagamento">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: white;
border-color: black;</string>
      </property>
      <item>
       <property name="text">
        <string>Pix</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cart√£o</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Boleto</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btnCadastra">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background-color: rgb(0, 85, 127);
font: 10pt &quot;Myanmar Text&quot;;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>CADASTRAR</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
